core/src/main/java/com/linecorp/armeria/client/IgnoreHostsTrustManager.java

```diff
@@ -49,7 +49,7 @@
 import com.google.common.collect.ImmutableSet;
 
 /**
- * An implementation of {@link X509ExtendedTrustManager} that skips verification on whitelisted hosts.
+ * An implementation of {@link X509ExtendedTrustManager} that skips verification on allowlisted hosts.
```

Comment:
There is no verb 'allowlist'. How about: the list of allowed hosts?

Reply:
Fixed